src/components/Dropzone.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback } from "react";
+import { useDropzone } from "react-dropzone";
+import styled, { css } from "styled-components";
+import { colors, shadows } from "../shared/constants";
+import { themeElement } from "../shared/theme";
+
+const dropStyles = css`
```

Comment:
Add `cursor: pointer;` to the dropzone styles.

Reply:
Done